lib/portlayer/storage/vsphere/toolbox_common.go

```diff
@@ -125,3 +128,34 @@ func isInvalidStateError(err error) bool {
 	}
 	return false
 }
+
+// IsToolBoxConflictErr checks for conflictError for online import
+func IsToolBoxStateChangeErr(err error) bool {
+	// check if error has to do with toolbox state changes
+	if soap.IsSoapFault(err) {
+		switch soap.ToSoapFault(err).VimFault().(type) {
+		case types.InvalidState:
+			return true
+		case types.InvalidPowerState:
+			return true
+		case types.GuestOperationsUnavailable:
+			return true
+		case types.SystemError:
+			return strings.Contains(err.Error(), VIX_E_TOOLS_NOT_RUNNING)
```

Comment:
Also disregard this if timestamps aren't included in the error string (I think they probably aren't).